### DISCUSSION TITLE
Sensitive Command Token Tracking Multiple Commands

### DISCUSSION THREAD

**Discussion by flizzer** [thinkst/canarytokens#253] [Feature Requests]
Greetings!  Love the theory behind Canary Tokens and the ease of implementation.  Was curious if there might there be any plans to enhance the Sensitive Command Token to be able to batch multiple commands in one token?  Perhaps it's a design choice to do one per command, but when considering implementing these, it would be easier to maintain/rollout if there was a way to batch up multiple commands.  Would love your thoughts on whether this is a viable feature or enhancement to add to the backlog.

Thanks and keep up the great work!

Brian

**Comment by jayjb** [comment]
Hi @flizzer,

Thanks for the idea (and feature request). Our team is currently chatting it through and we will get back to you with a response when we have one.
